### PR TITLE
Config params redesign

### PR DESCRIPTION
**Pull request by zpevma**
- pass whole config to each Section handler, each handler then parses the config in constructor
- [TODO ](https://github.com/csaf-tools/CVRF-CSAF-Converter/pull/49#discussion_r796785121) implemented

### CODE REVIEW

**Review by sustefil — APPROVED** [review]
LGTM.

I like the change that each SectionHandler picks whatever it needs from the config in the __init__, making it clear what config option is needed where

File: cvrf2csaf/config/config.yaml [hunk 1]
```diff
@@ -16,3 +16,5 @@ publisher_namespace: Publisher Namespace
 # Document Tracking
 force_update_revision_history: false
 
+# Document References
+force_default_reference_category: yes
```
Comment (tschmidtb51) [inline cvrf2csaf/config/config.yaml:20]:
```suggestion
force_insert_default_reference_category: true

**Review by tschmidtb51 — COMMENTED** [review]
In general, I approve. Minor naming detail and booleans should use true / false in the config to be consistent.

File: cvrf2csaf/config/config.yaml (hunk 1 above)
Comment (zpevma) [reply cvrf2csaf/config/config.yaml]: Sure, done.

**Review by tschmidtb51 — APPROVED** [review]
LGTM

**Pull request merged**